lib/features/auth/presentation/login/login_page.dart

```diff
@@ -1,275 +1,108 @@
-                                  const SizedBox(
-                                    width: 10,
-                                  ),
-                                  Expanded(
-                                    child: InkWell(
-                                      onTap: showForgetPasswordBottomSheet,
-                                      child: Text(
-                                        Strings.forgetPasswordQuestion,
-                                        overflow: TextOverflow.clip,
-                                        style: const TextStyle(
-                                          fontSize: 15,
-                                          color: Constant.violetColorDark,
-                                          fontFamily: AppFontFamily.regular,
-                                        ),
-                                      ),
-                                    ),
-                                  )
-                                ],
-                              ),
-                            ),
-                          ),
-                        ],
-                      ),
-                      Container(
-                        margin:
-                            const EdgeInsets.only(left: 16, right: 16, top: 32),
-                        child: SafeArea(
-                          child: FixitButton(
-                            enable: sendButtonEnable,
-                            loading: state is LoginLoading,
-                            label: Strings.login,
-                            background: Constant.violetColorDark,
-                            labelColor: Colors.white,
-                            onTab: onLogin,
+    return BlocProvider(
+      create: (context) => sl<LoginFormBloc>(),
+      child: Builder(
+        builder: (context) {
+          final formBloc = BlocProvider.of<LoginFormBloc>(context);
+          return Theme(
+              data: Theme.of(context).copyWith(
+                inputDecorationTheme: InputDecorationTheme(
+                  border: OutlineInputBorder(
+                    borderRadius: BorderRadius.circular(20),
+                  ),
+                ),
+              ),
+              child: Scaffold(
+                appBar: AppBar(title: const Text('Login Page')),
+                floatingActionButton: Column(
+                  mainAxisSize: MainAxisSize.min,
+                  children: <Widget>[
+                    FloatingActionButton.extended(
+                      heroTag: null,
+                      onPressed: formBloc.addErrors,
+                      icon: const Icon(Icons.error_outline),
+                      label: const Text('ADD ERRORS'),
+                    ),
+                  ],
+                ),
+                body: FormBlocListener<LoginFormBloc, String, String>(
+                  onSubmitting: (context, state) {
+                    //   LoadingDialog.show(context);
```

Comment:
Great work 👍

But you should, remove any unhelpful comments, to make your code more readable, and to reduce the size of the project!!